x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -378,8 +407,18 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps> = ({
   ]);
 
   const items = useMemo(
-    () => [...statusFilters, addEndpointExceptionComponent, addExceptionComponent],
-    [addEndpointExceptionComponent, addExceptionComponent, statusFilters]
+    () =>
+      !isEvent && ruleId
```

Comment:
Not sure if this is the right condition to allow add an event exception

Reply:
This should be fine for now as the Alerts table should only ever be displaying `event.kind:signal` documents (if the source document is `event.kind:event` that value is copied over to `original_event.kind:event`). That said, with the refactorings in flight around RAC I could see this logic changing to some sort of 'actionProvider' where the `document` and some `context` are used to determine the list of available actions. cc @andrew-goldstein and 
 @kqualters-elastic as they've been active in these Timeline changes.